src/psyclone/domain/lfric/transformations/lfric_loop_fuse_trans.py

```diff
@@ -127,7 +127,11 @@ def validate(self, node1, node2, options=None):
         # TODO #2498: access information for LFRic kernels do not have any
         # index information for field accesses, and the loop fusion dependency
         # tests will therefore fail. To avoid this, disable the dependency test
-        # in the generic loop fusion class for LFRic:
+        # in the generic loop fusion class for LFRic.
+        # TODO 257: if the loop fusion loop is implemented to just check that
```

Comment:
"loop fusion loop" should be "loop-fusion transformation"?

Reply:
Indeed.